File: backend/Entities/Models/PessoaFisica.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class PessoaFisica
{
    public Guid Id { get; set; }

    public string Cpf { get; set; } = null!;

    public virtual Pessoa IdNavigation { get; set; } = null!;

    public virtual PessoaFisicaComplemento? PessoaFisicaComplemento { get; set; }
}

File: frontend/src/app/Business/Models/pessoaFisica.ts
import { PessoaFisicaComplemento } from "./pessoaFisicaComplemento";


export interface PessoaFisica  {
    id: string;
    cpf: string;    
    pessoaFisicaComplemento: PessoaFisicaComplemento | null;
}